readme/template-readme.md

```diff
@@ -0,0 +1,101 @@
+# README Template
+
+*{Before using this template, read the accompanying README template guide}.*
+
+## {Project} logo and badges
+{This section is optional. Share your logo and add relevant badges to your README.}
+
+## {Project Name} 
+{Optional: Include the project URL and the project owner name}
+
+## Table of contents 
+{This section is optional.}
+1. Heading 1
+2. Heading 2
+3. Heading 3...
+
+## Project description 
+{Project description example available in README template guide}
+
+With *{Project Name}* you can *(verb)* *(noun)*...
+
+*{Project Name}* helps you *(verb)* *(noun)*....
+
+Unlike *(alternative)*, *{Project Name}* *(verb)* *(noun)*... 
+
+{Optional: Include screenshots and/or demo videos}
+
+## Who this project is for 
+This project is intended for {target audience} who want to {audience objective}. 
+
+## Project dependencies
+To use {Project Name}, you will need: 
+- Prerequisite 1 
+- Prerequisite 2
+- Prerequisite 3...
+
+## Instructions for using {Project Name}
+Get started with {Project Name} by {first step here}
+
+### How to get {Project Name}
+1. {Write the step here. Use a verb to start.}
+
+    {Explanatory text here}
```

Comment:
**suggestion:** I'm not sure, but I'd probably rephrase this into a full sentence for consistency.
Suggestions:
- {Explain the step here.}
- {Put your explanatory text here.}